components/pipeline-service/production/kflux-ocp-p01/deploy.yaml

```diff
@@ -1999,6 +2003,9 @@ spec:
                   value: "konflux-tenants"
                   effect: "NoSchedule"
             default-timeout-minutes: "120"
+        config-leader-election-resolvers:
```

Comment:
Why didn't we update the manifest for this cluster earlier?

Reply:
We might have forgot to do it, this is a new cluster that was just added, it was probably added by copy pasting another one and we forgot to regenerate the deploy.yaml